### PR TITLE
Add Alert, Avatar and Badge

### PR DESCRIPTION
**Pull request by Megha-Pathak**
image
image
image

### CODE REVIEW

File: Components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* Avatar */
+.avatar {
+    width: 60px;
+    height: 60px;
```
Comment (rkapoor10) [inline Components/avatar.css:4]: you may use rems here

File: Components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  
+  .av-md {
+    width: 60px;
+    height: 60px;
+  }
+  
+  .av-lg {
+    width: 90px;
+    height: 90px;
+  }
```
Comment (rkapoor10) [inline Components/avatar.css:25]: same as above

**Review by rkapoor10 — APPROVED** [review]
The code is well structured. You may use rems for larger units. 
Components look good to merge.

**Review by kotalwar-krishna — APPROVED** [review]
suggestion: be consistent using units (use rem )

**Pull request merged**